.gitignore

```diff
@@ -1,9 +1,9 @@
 bench_inv
 bench_ecdh
 bench_ecmult
+bench_schnorrsig
 bench_sign
 bench_verify
-bench_schnorr_verify
```

Comment:
Good thing this very dangerous oversight in the existing code is fixed!

Reply:
To the moon!